firebase.json

```diff
@@ -446,6 +451,11 @@
         "type": 301
       },
       {
+        "source": "/docs/dart-up-and-running/contents/ch02",
+        "destination": "/resources/books",
```

Comment:
Shouldn't the destination be /guides/language/language-tour, just like the next redirect?